### PR TITLE
docs: Fix WPF/GTK links

### PR DESCRIPTION
**Pull request by davkean**
These links were swapped.

### CODE REVIEW

File: doc/articles/getting-started/requirements.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ Uno Platform applications run on all macOS versions supported by Mac Catalyst -
 
 ## Linux
 
-Uno Platform applications run on Linux distributions and versions where .NET 7 is supported, [listed here](https://docs.microsoft.com/en-ca/dotnet/core/install/linux).
\ No newline at end of file
+Uno Platform applications run on Linux distributions and versions where .NET 7 is supported, [listed here](https://docs.microsoft.com/en-ca/dotnet/core/install/linux).
```
Comment (morning4coffe-dev) [inline doc/articles/getting-started/requirements.md:37]:
```suggestion
Uno Platform applications run on Linux distributions and versions where .NET 7 is supported, [listed here](https://learn.microsoft.com/dotnet/core/install/linux).
```

**Review by morning4coffe-dev — APPROVED** [review]

**Comment by morning4coffe-dev** [comment]
Thanks for the PR @davkean, can you please update the commit name to follow the https://www.conventionalcommits.org/en/v1.0.0/#summary guidelines, please?

**Review by Youssef1313 — APPROVED** [review]
Thanks for the fix @davkean!

Could you fix conventional commits validations?

**Comment by jeromelaban** [comment]
In some cases, it's possible for maintainers to rebase PRs from forks but not always... I wonder what allows for this, since ` Maintainers are allowed to edit this pull request.` is checked.

File: doc/articles/getting-started/requirements.md (hunk 1 above)
Comment (sasakrsmanovic) [reply doc/articles/getting-started/requirements.md:37]: @morning4coffe-dev  - this should be ".NET 7 or later. "

**Comment by jeromelaban** [comment]
@mergifyio squash

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

**Pull request merged**